### ISSUE TITLE
Level A issue - Subscribe Feed buttons do not read descriptive link text in Screen-reader Browse mode (NVDA & VoiceOver)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#822]
![image](https://user-images.githubusercontent.com/1927902/30550046-5b4e6b10-9c64-11e7-9ba2-d0f2f6cd9fcd.png)

When using Browse-mode (as opposed to be Focus-mode) with my screen-reader, the subscribe button link only reads out "link" instead of "Subscribe to [name of feed]". This affects any teaser list that has an RSS feed (e.g. News teaser list, Events teaser list, Features teaser list)

Issue occurs both in VoiceOver and NVDA.

**Why is this occurring:** The button only uses the title attribute to provide alternative text to the button. This works fine if you're in Focus mode, but the moment you're browsing the page in Browse mode (e.g. in NVDA, pressing the down arrow instead of the tab to navigate the page), you don't have access to the information in that title attribute.

![image](https://user-images.githubusercontent.com/1927902/30549656-55867494-9c63-11e7-9796-ace899ebe14d.png)

**Why does it matter:** We cannot guarantee users will be using Focus mode to navigate the page. They could be using Browse mode. I'm also not able to find a technique under either [WCAG 2.0 SC 1.1.1. Non-text Content](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=131%2C111#text-equiv-all) or [2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=131%2C111#navigation-mechanisms-refs) that allows us to use *just* the title attribute for alternative text, so this seems like a failure on both of those success criteria.

**How to fix:** Recommend putting text in the link and making that text only available to screen-readers.  This would be the same method that was used on the social media buttons.

![image](https://user-images.githubusercontent.com/1927902/30549869-e889b83c-9c63-11e7-902c-142be79be397.png)

**Comment by mmafe** [comment]
Fixed by #858
